cloud/scope/powervs_cluster.go

```diff
@@ -1977,3 +1969,26 @@ func (s *PowerVSClusterScope) isResourceCreatedByController(resourceType infrav1
 	}
 	return false
 }
+
+func (s *PowerVSClusterScope) bucketName() string {
```

Comment:
I see same functions in the machinescope as well, lets add a todo to get rid off these duplicate functions

Reply:
Done.